[torch/c10d] change class variable from private to protected (#149579)

GirasoleY opened the pull request:
Summary:


Change class variable from private to protected in ProcessGroupNCCL

Test Plan: Existing UT Pass.

Reviewed By: kingchc, kwen2501

Differential Revision: D71373067




cc @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o

GirasoleY commented:
Trigger EasyCLA check again.

GirasoleY commented:
/easycla

kwen2501 reviewed (approved):
LGTM. 
The lint issue is real. Can you please fix it?

GirasoleY committed:
[torch/c10d] change class variable from private to protected (#149645)

Summary:

Change class variable from private to protected in ProcessGroupNCCL

Test Plan: Existing UT Pass.

Reviewed By: kingchc, kwen2501

Differential Revision: D71373067
